src/model/manifest.go

```diff
@@ -10,8 +10,10 @@ import (
 )
 
 const (
-	SortById   = "id"
-	SortByPath = "path"
+	SortById       = "id"
+	SortByPath     = "path"
+	AllBranchesDef = "__all__"
```

Comment:
Pre internu reprezentaciu v `manifest.json` som zvolil nakoniec ine hodnoty, ... aby sa to neplietlo s tym co zadal uzivatel. Preto su upravene takto aj testy.